pages/common/croc.md

```diff
@@ -0,0 +1,32 @@
+# croc
+
+> Send and recieve files easily and security over any network.
+> More information: <https://github.com/schollz/croc>.
+
+- Send a file or folder:
+
+`croc send {{path/to/file}}`
+
+- Send a file or folder with a specific passphrase:
+
+`croc send --code {{your-own-created-passphrase}} {{path/to/file}}`
+
+- Recieve a file or folder on reciving machine:
+
+`croc {{the-passphrase-to-recieve-a-file}}`
+
+- Send over a custom relay and custom relay:
```

Comment:
This doesn't sound quite right. Maybe `to a custom relay`?

Reply:
Changed the text to:
```
Send and connect over a custom relay:
```
As it is `p2p` and the relay only helps with the setup. That should be  sufficient? :)